src/herder/HerderImpl.cpp

```diff
@@ -318,6 +318,10 @@ HerderImpl::broadcast(SCPEnvelope const& e)
 void
 HerderImpl::startRebroadcastTimer()
 {
+    if (mApp.getConfig().MANUAL_CLOSE && mApp.getConfig().RUN_STANDALONE)
```

Comment:
More generally, why do we do this at all if we are running with `RUN_STANDALONE`? There is no one to receive the broadcast.

Reply:
I wondered if it might be okay to change that regardless of `MANUAL_CLOSE`.  I've done so now.

Reply:
Actually, disabling it when only `RUN_STANDALONE` is present breaks, at least, the "3 nodes 2 running threshold 2"/"Over loopback" acceptance test.  I've pushed a change to revert to the original version which disables the timer only if both `MANUAL_CLOSE` and `RUN_STANDALONE` are configured.

Reply:
@MonsieurNicolas can you open an issue for that with some useful context about why it is like this? It strongly defies my expectations that we might need to broadcast when running standalone.

Reply:
Yeah @jonjove I opened https://github.com/stellar/stellar-core/issues/2719